blinkit-app/src/homePage/organisms/HomePage.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from "react";
+import vegetables, { total } from "../../DATA/ITEMS";
+import { UserProvider } from "../../userContext";
+import Foodbar from "../molecules/foodbar/Foodbar";
+import Footer from "../molecules/footer/Footer";
+import Header from "../molecules/header/Header";
+import MainPage from "../molecules/mainPage/MainPage";
+import Speciality from "../molecules/specialities/Speciality";
+import Information from "../molecules/information/Information"
+
+class HomePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: vegetables,
+      total: total,
+      query: "",
+      queryOutput: vegetables,
+    };
+  }
+  add = (index) => {
+    console.log(index);
+    vegetables[index].quantity = vegetables[index].quantity + 1;
```

Comment:
use optional chaining to avoid breaking
This could be written as 
vegetables[index].quantity <> vegetables[index]?.quantity